vendor/engines/projects/app/views/projects/shared/_select_project.html.haml

```diff
@@ -0,0 +1,9 @@
+.row
+  .span10
+    = f.input :project_id,
+      collection: order_detail.selectable_projects,
+      input_html: { class: "js--chosen", data: { placeholder: t(".placeholder") } },
+      selected: order_detail.project_id,
+      include_blank: true
+
+= javascript_include_tag "activate_chosen" if modal
```

Comment:
Could we do something like define a method in the JS for this, call it on load of that file to get the normal behavior, and trigger that same method call here rather than loading another file. I would be surprised if chosen didn't already handle having `chosen` called on it multiple times. Could we not deal with the `modal` check?